Documentation/docs/appendix.md

```diff
@@ -226,7 +226,12 @@ If the SIMPLE_EXTRUSION is used, bottom topography can be optionally added to th
 	\end{TOPOGRAPHY}
 
 ## Appendix E: ABAQUS mesh file format<a name="ABAQUS"></a>
-The ABAQUS mesh file which should use the extension `.inp` is divided into two parts. The top portion is the standard [Abaqus file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm). The skeleton of this form for a quadrilateral mesh with 122 corner nodes and 103 elements is as follows:
+
+The [Abaqus mesh file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm) is common in the finite element community. The ABAQUS mesh file should use the `.inp` extension. The standard Abaqus format includes a list corner data and list of element connectivity information. In this sense it is similar to the ISM format without high-order boundary information. The standard Abaqus format encodes a straight sided (linear) mesh.
```

Comment:
```suggestion
The [Abaqus mesh file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm) is common in the finite element community. The ABAQUS mesh file should use the `.inp` extension. The standard Abaqus format includes a list of nide data and list of element connectivity information. In this sense it is similar to the ISM format without high-order boundary information. The standard Abaqus format encodes a straight sided (linear) mesh.
```
Missing "of" and use of node (instead of corner) for consistency with later naming choices.

Reply:
Done. I also added a small note at the end about how the 3D boundary reordering is done.